qml/NarrationTimer.qml

```diff
@@ -21,7 +21,7 @@ import QtPositioning 5.3
 
 Timer {
     id: timer
-    interval: 3000
+    interval: 1000
```

Comment:
Its a good question - I did try to test it a bit and didn't see much difference. However, in general, it seems that Poor Maps does use considerable amount of CPU without any navigation, just when you follow your position. Interestingly, when I use Mapbox GL-based version with navigation, my device does not get worm as it does with the current Poor Maps. There is still CPU usage and I do have to look into comparison properly (note that the ambient temperature did change as well, so that will bias it towards later tests getting colder device even in the car).

The interval was set to get decent response time for voice commands (or directions as you suggested in other comment). My approach was to ensure that voice works as intended and, if needed, tune later.

Reply:
The current hack of putting tiles as MapQuickItems on top of the Map canvas is probably performace-wise a really bad idea. If the above change is negligible compared to the bigger issues, that would be enough to know.

Reply:
I did two short trips on the similar roads, but different directions. However, I think that the conditions were similar during the test. The reports were generated as soon as I stopped and we should consider the last two recordings as representative. In short, according to this test, the difference was not detected. On the basis of that, I suggest to keep interval equal to 1000

I am showing CPU usage and used frequency distribution. In addition, CPU sleep is shown to verify that device was kept alive during navigation.

## For interval 1000

![cpufreq_overview_](https://user-images.githubusercontent.com/15706605/31316890-43654a38-ac3f-11e7-8daf-07d38af106fd.png)
![cpu_overview](https://user-images.githubusercontent.com/15706605/31316891-43674dce-ac3f-11e7-990e-b2e960f9045e.png)
![cpustate_cpusleep](https://user-images.githubusercontent.com/15706605/31316892-43676f2a-ac3f-11e7-92c7-39dc6bd38453.png)

## For interval 3000

![cpufreq_overview_](https://user-images.githubusercontent.com/15706605/31316896-5c132078-ac3f-11e7-87e0-d9f81cf55d8f.png)
![cpu_overview](https://user-images.githubusercontent.com/15706605/31316897-5c15f212-ac3f-11e7-83c0-7c5f8745a91f.png)
![cpustate_cpusleep](https://user-images.githubusercontent.com/15706605/31316898-5c16ffcc-ac3f-11e7-8460-2b65e99bcb73.png)

Reply:
OK, let's keep the 1000.